doc/introduction/pulse.rst

```diff
@@ -0,0 +1,289 @@
+
+    f1 = pwc(timespan=(2, 7))  # TODO: maybe this should be renamed window for uniformity?
+    H = f1 * qml.PauliX(0)
+
+    # passing pwc((2, 7)) an array evenly distributes the array values in the interval t=2 to t=7
+    H(params=[[1, 2, 3, 4, 5]], t=2.3)
+    >>> 1.0*(PauliX(wires=[0]))
+
+    # different time, same bin, same result
+    H(params=[[1, 2, 3, 4, 5]], t=2.5)
+    >>> 1.0*(PauliX(wires=[0]))
+
+    # next bin
+    H(params=[[1, 2, 3, 4, 5]], t=3.1)
+    >>> 2.0*(PauliX(wires=[0]))
+
+    # outside the window where the function is assigned non-zero values
+    H(params=[[1, 2, 3, 4, 5]], t=8)
+    >>> 0.0*(PauliX(wires=[0]))
+
+ParametrizedEvolution
+---------------------
+During a pulse sequence spanning time :math:`(t_0, t_1)`, the state evolves according to the time-dependent Schrodinger equation
+
+.. math:: \frac{\partial}{\partial t} |\psi\rangle = -i H(t) |\psi\rangle
+
+realizing a unitary evolution :math:`U(t_0, t_1)` of the input state, i.e.
+
+.. math:: |\psi(t_1)\rangle = U(t_0, t_1) |\psi(t_0)\rangle
+
+A :class:`~.ParametrizedEvolution` is this solution :math:`U(t_0, t_1)` to the time-dependent
+Schrodinger equation for a :class:`~.ParametrizedHamiltonian`.
+
+Creation of the :class:`~.ParametrizedEvolution` uses an a numerical ordinary differential equation
+solver (`here <https://github.com/google/jax/blob/main/jax/experimental/ode.py>`_). The :class:`~.ParametrizedEvolution`
+can be found using the :func:`~qml.evolve` function:
+
+.. code-block:: python
+
+    from jax import numpy as jnp
+
+    f1 = lambda p, t: p * jnp.sin(t) * (t - 1)
+    H = 2 * qml.PauliX(0) + f1 * qml.PauliY(1)
+    ev = qml.evolve(H)
+    ev
+    >>> ParametrizedEvolution(wires=[0, 1])
+
+The initial :class:`~.ParametrizedEvolution` does not have parameters defined, and so will
+not have a defined matrix. To complete initialization of the :class:`~.Operator`, it must be passed
+parameters and a time interval:
+
+.. code-block:: python
+
+    ev([1.2], t=[0, 4]).matrix()
+    >>> Array([[-0.14115842+0.j        ,  0.03528605+0.j        ,
+                 0.        -0.95982337j,  0.        +0.23993255j],
+               [-0.03528605+0.j        , -0.14115842+0.j        ,
+                 0.        -0.23993255j,  0.        -0.95982337j],
+               [ 0.        -0.95982337j,  0.        +0.23993255j,
+                -0.14115842+0.j        ,  0.03528605+0.j        ],
+               [ 0.        -0.23993255j,  0.        -0.95982337j,
+                -0.03528605+0.j        , -0.14115842+0.j        ]],      dtype=complex64)
+
+
+The parameters can be updated by calling the :class:`~.ParametrizedEvolution` again with different inputs.
+
+Additional options with regards to how the matrix is calculated can be passed to the :class:`.ParametrizedEvolution`
+along with the parameters, as keyword arguments:
+
+.. code-block:: python
+
+    qml.evolve(H)(params=[1.2], t=[0, 4], atol=1e-6, mxstep=1)
```

Comment:
Is `mxstep` JAX lingo? seems weird to me to skip the "a"...

Reply:
Yes, I don't know why they did that, but they did